int32 tag_id
CameraPoint[] corners